conans/client/toolchain/msbuild.py

```diff
@@ -32,14 +34,20 @@ def command(self, sln):
                                 vcvars_ver=None)
         cmd = ('%s && msbuild "%s" /p:Configuration=%s /p:Platform=%s '
                % (vcvars, sln, self.build_type, self.platform))
+
+        if self.compiler == 'intel':
```

Comment:
From some parts of the docs (https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-cpp-compiler-for-windows/top.html), it could seem that the intel_compilervars already include setting up the Visual environment too, and then, calling vcvars explicitly would not be needed? I guess you have already checked, but just in case.

Reply:
yes, it works, I'll change code a bit to avoid redundant vcvars

Reply:
oh, too late :) never mind then

Reply:
see #8078